Resources/Private/ModifyDataWithCommandCKEditor/src/modifyDataPlugin.js

```diff
@@ -0,0 +1,20 @@
+import Command from '@ckeditor/ckeditor5-core/src/command';
```

Comment:
You should use the exposed api classes from Neos instead of referencing the classes form ck itself.
This way they are not duplicated in your bundle.

```suggestion
import {Command, Plugin} from 'ckeditor5-exports';
```